migration/src/m0000335_create_version_scheme.rs

```diff
@@ -0,0 +1,193 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        // Replace the sample below with your own migration scripts
+        manager
+            .create_table(
+                Table::create()
+                    .table(VersionScheme::Table)
+                    .col(ColumnDef::new(VersionScheme::Id).string().primary_key())
+                    .col(ColumnDef::new(VersionScheme::Name).string().not_null())
+                    .col(ColumnDef::new(VersionScheme::Description).string())
+                    .to_owned(),
+            )
+            .await?;
+
+        let db = manager.get_connection();
+
+        insert(
+            db,
+            "semver",
+            "Semantic Versioning",
+            Some("Semantic versioning as defined by SemVer 2.0.0 (see https://semver.org/)"),
+        )
+        .await?;
+
+        insert(
+            db,
+            "ecosystem",
+            "Ecosystem-specific",
+            Some("Ecosystem-specific versioning; otherwise unspecified"),
+        )
+        .await?;
+
+        insert(
+            db,
+            "git",
+            "Git commit-hash",
+            Some("Git commit-hash-based versioning"),
+        )
+        .await?;
+
+        // package-url `vers` variants
+        //
+        // https://github.com/package-url/purl-spec/blob/version-range-spec/VERSION-RANGE-SPEC.rst
+
+        insert(
+            db,
+            "deb",
+            "Debian and Ubuntu",
+            Some("https://www.debian.org/doc/debian-policy/ch-relationships.html"),
+        )
+        .await?;
+
+        insert(
+            db,
+            "rpm",
+            "RPM distributions",
+            Some("https://rpm-software-management.github.io/rpm/manual/dependencies.html"),
+        )
+        .await?;
+
+        insert(
+            db,
+            "gem",
+            "Rubygems",
+            Some("https://guides.rubygems.org/patterns/#semantic-versioning"),
+        )
+        .await?;
+
```

Comment:
These make less sense as a rusty enum, I suppose, but it still feels kinda gross hardcoding things in SQL rather than Rust.

Reply:
This could be rusty, but the SQL also has to know the name of the version scheme to select the correct `cmp` function.